app/metal-controller-manager/internal/api/api.proto

```diff
@@ -42,11 +43,14 @@ message Address {
 message CreateServerResponse {
   bool wipe = 1;
   bool insecure_wipe = 2;
+  double reboot_timeout = 3;
 }
 
 message MarkServerAsWipedRequest { string uuid = 1; }
+message WipeHeartbeatRequest { string uuid = 1; }
```

Comment:
I think we should keep this generic and just call it `Heartbeat`. I can see the heartbeat being useful outside of wiping.